core/parachain/pvf/session_params.hpp

```diff
@@ -12,18 +12,23 @@
 namespace kagome::parachain {
   inline outcome::result<runtime::RuntimeContext::ContextParams> sessionParams(
       runtime::ParachainHost &api, const primitives::BlockHash &relay_parent) {
+    constexpr uint32_t kDefaultHeapPagesEstimate = 32;
+    constexpr uint32_t kExtraHeapPages = 2048;
```

Comment:
Are there any reference values for constants in polkadot-sdk? If so, I suggest to comment with the URL to them.